### PR TITLE
refactor: ZeroScanner (R-Hero) does not use Travis API and uses GitHub scanner 

### PR DESCRIPTION
**Pull request by javierron**
- Zero scanner (R-Hero)
  - update: Remove usage of Travis API and use GitHub scanner instead.
- GitHub scanner: 
  - fix: use UTC timezone
  - update: fetch commit data in parallel

### CODE REVIEW

File: src/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/ZeroScanner.java [hunk 1]
```diff
@@ -53,18 +52,20 @@ static void setup(){
         RepairnatorConfig.getInstance().setGithubToken(System.getenv("GITHUB_OAUTH"));
 
         //pipeline image tag
-        RepairnatorConfig.getInstance().setDockerImageName("repairnator/pipeline:latest");
```
Comment (monperrus) [inline src/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/ZeroScanner.java:56]: let's keep the latest

**Review by monperrus — APPROVED** [review]

**Comment by monperrus** [comment]
@javierron rebase to get CI green?

**Comment by javierron** [comment]
Rebased to latest version. Ready to merge if CI is green :smile:

@monperrus

**Comment by monperrus** [comment]
and CI green here as well, super to happy to complete this pending PR! Thanks @javierron

**Pull request merged**